test/Npgsql.Tests/CommandTests.cs

```diff
@@ -265,27 +265,23 @@ public async Task Cancel()
         }
 
         [Test, Description("Cancels an async query with the cancellation token")]
-        [Ignore("Flaky on CoreCLR")]
         public async Task CancelAsync()
         {
-            var cancellationSource = new CancellationTokenSource();
+            if (IsMultiplexing)
+                Assert.Ignore("Async cancellation isn't supported with multiplexing yet.");
+
             using (var conn = await OpenConnectionAsync())
-            using (var cmd = CreateSleepCommand(conn))
+            using (var cmd = CreateSleepCommand(conn, 5))
```

Comment:
Small note: to make the test less sensitive to timing issues and flakiness, it may be better to make this a really high value (this is why CreateSleepCommand does 1000 seconds by default) and to have a timeout. I've seen CI VMs get stuck for a few seconds sometimes, this would trigger a failure here.